src/crispy/StackTrace.cpp

```diff
@@ -54,11 +56,11 @@ constexpr size_t MAX_FRAMES { 128 };
 constexpr size_t SKIP_FRAMES { 0 };
 
 #if defined(__linux__) || defined(__APPLE__)
-struct Pipe // {{{
+struct pipe_wrap // {{{
```

Comment:
```suggestion
struct system_pipe // {{{
```

that's probably a better name than `pipe_wrap`